src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/CheckboxAccessibility.h

```diff
@@ -28,5 +28,6 @@
 @interface CheckboxAccessibility : ButtonAccessibility <NSAccessibilityCheckBox> {
 
 };
-- (id)accessibilityValue;
+- (NSAccessibilityRole _Nonnull)accessibilityRole;
+- (id _Nonnull)accessibilityValue;
```

Comment:
Looks like the value can be null. No?

Reply:
Not here. The value here is a state of the checkbox - checked or not checked, unlike text components where value can be set to null, here value is being generated from the component state and can not be null.